src/assets/drizzle/styles/components/_footer.scss

```diff
@@ -0,0 +1,18 @@
+.#{$app-namespace}-c-footer {
+  background-color: $purp;
+  padding: $space-inset-short-l;
+}
+
+.#{$app-namespace}-c-footer__icons {
+  @media (max-width: $footer-collapse-breakpoint) {
+    margin-top: $space-s;
+  }
+}
+
+.#{$app-namespace}-c-footer__link {
+  display: inline-block;
+}
+
+.#{$app-namespace}-c-footer__link:hover {
+  color: $highlight;
```

Comment:
Can you confirm that this orange on purple passes AA compliance?

Reply:
This combo only passes for **Large Text AA**. But Font size is 16px so no dice. I could make the link bigger if all we need is  **Large Text AA**. Otherwise I can look at some other combos. 


![screen shot 2018-10-08 at 3 42 49 pm](https://user-images.githubusercontent.com/181652/46630185-73a6e480-cb11-11e8-93c8-6e0a4a2c77fa.png)

Reply:
I'd vote for a diff color combo. Would a lighter orange work? Because that purple is sweet.

Reply:
How do you feel about the `$red-light` from Spark colors?

![screen shot 2018-10-09 at 9 58 39 am](https://user-images.githubusercontent.com/181652/46674647-b1a41700-cbaa-11e8-9980-bdb676951bfa.png)

Reply:
@motionsuggests Spark colors are in the midst of seasonal change currently, So i think keeping it separate from that is best.